lib/index.js

```diff
@@ -58,7 +58,8 @@ var epilogue = {
       pagination: args.pagination,
       updateMethod: this.updateMethod,
       search: args.search,
-      sort: args.sort
+      sort: args.sort,
+      reloadOnCreate: args.reloadOnCreate
```

Comment:
@mbroadst Upon reflection, I think for data consistency, the Create and Update must meet the same and consistently. So the reload must be default to TRUE and if this is the wish to make data processing regardless back then in that case, disable the reload is good for performance. But you have to have conscious that the return of the Update will be inconsistent and that the return of partial received situation will create.

I want modified like that by default:
```
reload: {
  onCreate: true,
  onUpdate: true
}
```